pkg/metrics/instrumenting.go

```diff
@@ -7,7 +7,6 @@ import (
 	"runtime"
```

Comment:
In the new version there were breaking changes again compared to the previous try, but all figured.